### PR TITLE
samples: adds speech export to gcs sample

### PR DESCRIPTION
**Pull request by b-loved-dreamer**
Sample of speech export to GCS

### CODE REVIEW

File: samples/snippets/speech_to_storage_beta.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    operation = speech_client.long_running_recognize(request=request)
+
+    print("Waiting for operation to complete...")
+    response = operation.result(timeout=90)
+
+    # Each result is for a consecutive portion of the audio. Iterate through
+    # them to get the transcripts for the entire audio file.
+    for result in response.results:
+        # The first alternative is the most likely one for this portion.
+        print(u"Transcript: {}".format(result.alternatives[0].transcript))
```
Comment (busunkim96) [inline samples/snippets/speech_to_storage_beta.py:55]:
```suggestion
        print("Transcript: {}".format(result.alternatives[0].transcript))
```

All text is unicode in Python 3, so the `u'` is not needed. https://docs.python.org/release/3.0.1/whatsnew/3.0.html

Comment (b-loved-dreamer) [reply samples/snippets/speech_to_storage_beta.py]: Thank you @busunkim96!

File: samples/snippets/speech_to_storage_beta.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    output_config = speech.TranscriptOutputConfig(gcs_uri=output_storage_uri)
+
+    # Speech configuration object
+    config = speech.RecognitionConfig(
+        encoding=encoding,
+        sample_rate_hertz=sample_rate_hertz,
+        language_code=language_code,
+    )
+
+    # Package the long-running request
```
Comment (telpirion) [inline samples/snippets/speech_to_storage_beta.py:38]: NIT: I would a different verb than "package." Maybe "compose" or "create."

**Review by telpirion — APPROVED** [review]

File: samples/snippets/speech_to_storage_beta.py (hunk 2 above)
Comment (b-loved-dreamer) [reply samples/snippets/speech_to_storage_beta.py]: Thank you @telpirion!

**Pull request merged**